### PR TITLE
Allow state that is boolean to be toggled

### PR DESCRIPTION
**Pull request by joecritch**
Folks! I've added a checkbox for boolean state, e.g.

👍 Saves having to manually type "true/false"

screen shot 2017-03-22 at 21 17 30

It's my first PR on this repo, so nit away.

### CODE REVIEW

File: frontend/DataView/DataView.js [hunk 1]
```diff
@@ -306,6 +323,13 @@ var styles = {
     top: 4,
   },
 
+  toggler: {
+    marginLeft: -12,
```
Comment (jaredly) [inline frontend/DataView/DataView.js]: if we're already doing `position: absolute`, might as well just do `left: -12` here, to align with `top: -1`. And we probably don't need `paddingRight: 3`?
In fact, I don't think the `padding` will have any effect on a checkbox

**Comment by jaredly** [comment]
Neat idea!

**Comment by joecritch** [comment]
@jaredly Thanks man. Fixed up the styling as per your suggestion. Also, I've ensured the checkbox is a controlled component. (I was passing in something that was evaluating as `undefined`, so therefore wasn't being controlled. Doh!)

**Comment by jaredly** [comment]
solid!

**Pull request merged**